tests/support.py

```diff
@@ -2,17 +2,37 @@
 import sys
 import unittest
 import platform
+from wsgiref.validate import validator
+
+from gunicorn import __version__
 
 HOST = "127.0.0.1"
 
 
+@validator
+def app(environ, start_response):
+    """Simplest possible application object"""
+
+    data = b'Hello, World!\n'
+    status = '200 OK'
+
+    response_headers = [
+        ('Content-type', 'text/plain'),
+        ('Content-Length', str(len(data))),
+        ('X-Gunicorn-Version', __version__),
```

Comment:
We can drop the X-Gunicorn-Version header.

Reply:
also the commit message should be  "remove Cyrillic chars" this has nothing to do with "russian" .